discovery/gce/gce.go

```diff
@@ -178,6 +178,15 @@ func (d *Discovery) refresh(ctx context.Context) ([]*targetgroup.Group, error) {
 			addr := fmt.Sprintf("%s:%d", priIface.NetworkIP, d.port)
 			labels[model.AddressLabel] = model.LabelValue(addr)
 
+			// Append additional interface private IP address metadata if multiple interfaces exist.
+			if len(inst.NetworkInterfaces) > 1 {
+				gceLabelEthAddressPrefix := gceLabel + "private_ip_eth"
+				for idx, iface := range inst.NetworkInterfaces {
+					gceLabelEthAddressPrivate := model.LabelName(fmt.Sprintf("%s%d", gceLabelEthAddressPrefix, idx))
+					labels[gceLabelEthAddressPrivate] = model.LabelValue(iface.NetworkIP)
```

Comment:
This can be simplified.

```suggestion
				for idx, iface := range inst.NetworkInterfaces {
					gceLabelNetAddress := model.LabelName(fmt.Sprintf("%sinterface_ipv4_eth%d", gceLabel, idx))
					labels[gceLabelNetAddress] = model.LabelValue(iface.NetworkIP)
```

Reply:
@roidelapluie `..._network_ipv4_eth...` or `..._interface_ipv4_eth...`?